src/pages/specimen-collection.js

```diff
@@ -277,7 +277,6 @@ const SpecimenCollection = () => {
                   <Stack gap={7}>
                     <RadioButtonGroup
                       orientation="vertical"
-                      legendText="Type"
```

Comment:
Instead of removing legendText. It is preffered to set it to empty string since it is required for this component (CDS guidelines). Take a look at the browser console there is a warning

```js
legendText=""
```